test/manual/k8s_crds/deploy

```diff
@@ -1,11 +1,79 @@
 #!/bin/bash -e
 
-CURRENT_DIR=$(dirname "$0")
+function pod_ready() {
+  kubectl describe pod ${POD_NAME} | awk '/Ready/{if ($2 != "True") exit 1}'
+}
 
-echo "Stopping deployment..."
-./stop_deployment || true
+function http_proxy_resp() {
+  POD_NAME=$(kubectl get po -l app=secretless-k8s-test -o jsonpath="{.items[0].metadata.name}")
+  kubectl exec -it ${POD_NAME} -c echo-server -- \
+    env http_proxy=localhost:8000 curl -s localhost:8080
+}
+function http_resp() {
+  POD_NAME=$(kubectl get po -l app=secretless-k8s-test -o jsonpath="{.items[0].metadata.name}")
+  kubectl exec -it ${POD_NAME} -c echo-server -- curl --connect-timeout 4 -s localhost:8080 &> /dev/null
+}
 
-echo "Deploying..."
-kubectl apply -f "$CURRENT_DIR/deployment.yaml"
+CURRENT_DIR=$(dirname $0)
 
-echo "Deployment done!"
+echo "cleaning up previous deployments"
+./stop_deployment &> /dev/null || true
+echo "cleaned"
+echo ""
+
+echo "deploying CRDs"
+kubectl apply -f $CURRENT_DIR/deployment.yaml &> /dev/null
+echo "deployed"
+echo ""
+
+echo "waiting for CRDs to be ready"
+until $(kubectl get configurations.secretless.io &> /dev/null); do
```

Comment:
My manual test was stuck in this loop, I suspect because the CRD was not installed.